stdlib/public/TensorFlow/Ops.swift

```diff
@@ -663,30 +663,50 @@ public extension Tensor {
   }
 }
 
-public extension Tensor {
-  /// Concatenates tensors along the first dimension.
-  /// - Precondition: The tensors must have the same shape, except for the
-  ///   leading dimension.
-  @inlinable @inline(__always)
-  func concatenated(with other: Tensor) -> Tensor {
-    return Raw.concatV2([self, other], axis: Tensor<Int32>(0))
-  }
 
+@inlinable @inline(__always)
+public func _splitV<T: TensorFlowScalar, Tidx: BinaryInteger & TensorFlowScalar>(
```

Comment:
This helper function shouldn't be public. Is there no raw op for this? Use the `#tfop` directly in the VJP's body.

Reply:
No raw op, returns a variable number of tensors...